src/lib.rs

```diff
@@ -136,15 +136,14 @@ impl<'a, NetVal: NetworkValidation, T> Uri<'a, NetVal, T> {
 
 impl<'a, T> Uri<'a, bitcoin::address::NetworkUnchecked, T> {
     /// Checks that the bitcoin network in the URI is `network`.
```

Comment:
Actually, this comment was wrong as it suggests checking equality which would be wrong.

I'd prefer to update it before releasing next version.